QuizApp/QuizListCollectionViewCell.swift

```diff
@@ -0,0 +1,157 @@
+import Foundation
+import UIKit
+import SnapKit
+import Kingfisher
+
+class QuizListCollectionViewCell: UICollectionViewCell {
+
+    static let id = "QuizListCollectionViewCell"
+
+    private var quiz: QuizModel!
+    private var quizImageView: UIImageView!
+    private var quiztitleLabel: UILabel!
+    private var descriptionLabel: UILabel!
+    private var difficultyView1: UIView!
+    private var difficultyView2: UIView!
+    private var difficultyView3: UIView!
+    private var difficultyStackView: UIStackView!
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        contentView.backgroundColor = UIColor(red: 1, green: 1, blue: 1, alpha: 0.3)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func set(for quiz: QuizModel) {
+        self.quiz = quiz
+        buildViews()
```

Comment:
Call `buildViews` in init. Only what is dependent on QuizModel should be set in this func.